CHANGELOG.md

```diff
@@ -108,7 +112,7 @@
 
 ### Various fixes & improvements
 
-- ref: Pin cocoapods version  (#496) by @brustolin
+- ref: Pin cocoapods version (#496) by @brustolin
```

Comment:
please revert the other (unrelated, cosmetic) edits to this file -- please make these in a separate PR if you want